### ISSUE TITLE
Test failure message about redirection

### ISSUE THREAD

**Issue by lwalik** [Lowgular/testgular#5]
Video: Button -> Expect Url Change after Click 
Description: 
When we perform a test where we expect a redirect to a specific page when the button is pressed and the test fails, we wait a certain amount of time and then a message appears. Is it possible for the error message to tell what the reason for the failure is (missing button, redirection to the wrong page, lack of redirection)? I think this could be helpful. From the test level, do we want to have this information?

**Comment by gradzio** [comment]
yes we should have a better error messaging, let me try to prototype it here as well
